src/platform/Ameba/BUILD.gn

```diff
@@ -71,4 +71,14 @@ static_library("Ameba") {
       "AmebaOTAImageProcessor.h",
     ]
   }
+
+  if (chip_crypto == "platform") {
```

Comment:
Do you actually need platform crypto if you're building `mbedtls` anyway? I mean even if you use `chip_crypto = "mbedtls"`, you can still inject a custom operational keystore.

Reply:
You are right, its not necessary to utilize platform crypto in this case. I have made the changes.